assets/dev/js/editor/elements/views/container.js

```diff
@@ -57,6 +57,8 @@ const ContainerView = BaseElementView.extend( {
 			directionSettingKey = this.getDirectionSettingKey() + deviceSuffix,
 			currentDirection = this.container.settings.get( directionSettingKey );
 
+		this.updateNeedHelpLink();
```

Comment:
Hey @ronenelementor Nice investigation, but this is unfortunately not the correct place to add this code.

There is a JS hook that runs when the panel opens: `panel/open_editor/container`, you can listen to this hook and update the link from there. This hook and callback can be added in `grid-container.js`

Then you can also update `views/container.js` and remove `this.updateNeedHelpLink()` in the `onRender()` method, it is not needed as far as I can see.

So now we covered all scenarios:
1) When the panel opens
2) When the Container Type setting is changed.

I will attach screenshots of the proposed diff, but please test it thoroughly afterwards to make sure everything is covered in the mind map (DnD, refresh, setting change, import etc).

![Screenshot 2023-12-11 at 10 31 46](https://github.com/elementor/elementor/assets/20996103/19a23019-3833-4d03-9143-024fc5fc8d99)

![Screenshot 2023-12-11 at 10 32 15](https://github.com/elementor/elementor/assets/20996103/0bea462a-a079-40c3-a93d-b610eed2ee73)

Reply:
Will do! Thanks!